Packs/Gurucul/ReleaseNotes/1_1_1.md

```diff
@@ -0,0 +1,39 @@
+#### Incident Fields
+- Updating Gra Case Status
+- Added Gra Anomalies
+- Added Gra Anomaly Name
+- Added Gra Anomaly Resource Name
+- Added Gra Anomaly Risk Score
+- Added Gra Anomaly Status
+- Added Gra Assignee
+- Added Gra Assignee Type
+- Added Gra Risk Accepted Date
+
+#### Integrations
+##### Gurucul-GRA
+
+- **New drop-down** ***Fetch Incident Cases*** **is added in** ***Add instance***. **The valid options are:**
+  - ***Case Per Anomaly***: Incident is created per Anomaly.
+  - ***Case For All Anomalies***: Incident is created per Case.
+- **Bug Fixes**:
+  - Fixed an issue with fetch incidents, which caused incident duplication.
+  - Fixed an issue with fetch incidents, which caused empty/blank incident.
+  
+#### Classifiers
+##### Gurucul-GRACases - Classifier
+- Added Gra Anomalies
+- Added Gra Anomaly Name
+- Added Gra Anomaly Resource Name
+- Added Gra Anomaly Risk Score
+- Added Gra Anomaly Status
+- Added Gra Assignee
+- Added Gra Assignee Type
+- Added Gra Risk Accepted Date
+
+#### Mappers
+##### Gurucul-GRACases Incoming Mapper
+- Name Changed
```

Comment:
i think the release notes here are wrong, need to mention the added fields

Reply:
Pushed changes